x-pack/plugins/spaces/server/spaces_client/spaces_client.ts

```diff
@@ -130,10 +136,19 @@ export class SpacesClient implements ISpacesClient {
       );
     }
 
+    if (this.isServerless && space.hasOwnProperty('solution')) {
```

Comment:
Create/update endpoint is not registered right now in serverless, but we plan to enable it. Refer to [this comment](https://github.com/elastic/kibana/issues/183559#issuecomment-2117699399), that's why validation is needed here.